honeybee/radiance/datatype.py

```diff
@@ -4,20 +4,57 @@
 import warnings
 import os
 
-__all__ = ['RadDescrPath','RadDescrNum','RadDescrBoolFlags','RadDescrNumTuple']
+__all__ = ['RadiancePath', 'RadianceNumber', 'RadianceBoolFlag',
+           'RadianceNumericTuple']
 
-class RadDescrDefault(object):
+
+class RadianceDefault(object):
     """
-    The default descriptor which defines the __get__ method. Since the get
-    method is not likely to be fancy most descriptors can inherit this class.
+    The default descriptor for radiance commands. Provides base case attributes
+    for other descriptors.
+
+    Attributes:
+        _name: Required for all cases. Name of the flag, like 'ab' for '-ab 5'
```

Comment:
@sariths I think we're talking about two different things. I was mentioning _ in docstring and not the code. The arguments are ```(name, descriptiveName=None, expectedInputs=None, validRange=None)``` but in docstring you're using ```_name```, ```_descriptiveName```, etc.